std/internal/math/biguintcore.d

```diff
@@ -344,6 +344,17 @@ public:
         return buff[z-frontExtraBytes..$];
     }
 
+    /**
+     * Convert to an octal string.
+     */
+    char[] toOctalString() const
+    {
+        auto predictLength = 8*BigDigitBits / 3;
```

Comment:
Good catch, I wasn't paying attention and failed to actually multiply by `data.length`. Fixed.